frontend/src/components/WebBrowser/websites/MyPayment/index.js

```diff
@@ -61,8 +61,8 @@ class MyPayment extends React.Component {
           render={() => (
             <LoginContainer>
               <h1>
-                Thank you for checking. Your recent transactions have been
-                verified.
+                Thank you for reaching out to the Money Transfer service. We'll
+                be in touch shortly to arrange your payment.
               </h1>
             </LoginContainer>
           )}
```

Comment:
Any ideas why github doesn't like this?

Reply:
@bendiggle I think because `We'll` uses a quote as an apostrophe `'`. Try using a curly quote instead: `’` (https://practicaltypography.com/straight-and-curly-quotes.html)

Reply:
@jkimbo That seems to have fixed it! Cheers!